src/embed-code/embed-code-service.ts

```diff
@@ -37,6 +38,26 @@ export class EmbedCodeService {
 			});
 		}
 	}
+	public static async updateEmbedCode(data: EmbedCode): Promise<void> {
+		let url: string | undefined = undefined;
+
+		try {
+			url = `${getEnv('PROXY_URL')}/embed-codes/${data.id}`;
+
+			await fetchWithLogoutJson<{
+				message: 'success';
+			}>(url, {
+				method: 'POST',
+				body: JSON.stringify(data),
+				forceLogout: false,
+			});
+		} catch (err) {
+			throw new CustomError('Failed to create embed code', err, {
+				url,
+				data,
+			});
```

Comment:
might be helpful to log the error for developers, since the error could be swallowed by some try catch higher up the throw chain
```suggestion
			const error = new CustomError('Failed to create embed code', err, {
				url,
				data,
			});
			console.error(error);
			throw error;
```